Samples/WinMLSamplesGallery/WinMLSamplesGallery/Common/SamplesGrid.xaml

```diff
@@ -0,0 +1,49 @@
+<Page
+    x:Class="WinMLSamplesGallery.SamplesGrid"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:WinMLSamplesGallery"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
```

Comment:
are you using the blend/2008 namespace? same for others here. let's remove the ones we're not using.

Reply:
Done.